userguide/install.rst

```diff
@@ -807,10 +807,6 @@ configuration. The system can be recovered with a few steps:
    into an older version of the operating system is performed,
    a configuration created in a later version cannot be restored.
 
-#ifdef freenas
-#include snippets/upgradingazfspool.rst
```

Comment:
If this text is obsolete or no longer used, should we remove the file too?

Reply:
Must have been something I was investigating, deleted, still needed it, but forgot to replace it. I have put it back in. It appears it is still needed.